Library/Formula/libxslt.rb

```diff
@@ -17,8 +17,17 @@ class Libxslt < Formula
 
   depends_on 'libxml2'
 
+  head do
+    patch :DATA
+    url 'git://git.gnome.org/libxslt'
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
   def install
-    system "./configure", "--disable-dependency-tracking",
+    cmd = if build.head? then "./autogen.sh" else "./configure" end
```

Comment:
Can we call `autogen.sh` first and then `./configure` afterwards? Might be a bit more elegant.